cpp/src/Ice/PropertyNames.h

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) ZeroC, Inc.
 
-// Generated by makeprops.py from PropertyNames.xml, Fri Sep 27 14:25:19 2024
+// Generated by makeprops.py from PropertyNames.xml, Tue Oct  1 15:07:27 2024
```

Comment:
Can you remove the timestamp?

Reply:
Likely git will still update the file as it will be "touched" when regenerated.

Reply:
> Likely git will still update the file as it will be "touched" when regenerated.

I doubt it, git doesn't care about timestamps just track changes

Reply:
Cool! I didn't know this.